update-resources.sh

```diff
@@ -40,171 +40,173 @@ function chkfs() {
+  wget -q -O ./windows/winPEAS/winPEASx86_ofs.exe "https://github.com/carlospolop/PEASS-ng/releases/latest/download/winPEASx86_ofs.exe"
 }
 
 function add_linux_smart_enumeration() {
   info "Downloading Linux Smart Enumeration"
   chkfs "./linux/"
-  wget -nv -O ./linux/linux-smart-enumeration.sh "https://github.com/diego-treitos/linux-smart-enumeration/raw/master/lse.sh"
+  wget -q -O ./linux/linux-smart-enumeration.sh "https://github.com/diego-treitos/linux-smart-enumeration/raw/master/lse.sh"
 }
 
 function add_linenum() {
   info "Downloading LinEnum"
   chkfs "./linux/"
-  wget -nv -O ./linux/LinEnum.sh "https://raw.githubusercontent.com/rebootuser/LinEnum/master/LinEnum.sh"
+  wget -q -O ./linux/LinEnum.sh "https://raw.githubusercontent.com/rebootuser/LinEnum/master/LinEnum.sh"
 }
 
 function add_linux_exploit_suggester() {
   info "Downloading Linux Exploit Suggester"
   chkfs "./linux/"
-  wget -nv -O ./linux/linux-exploit-suggester.sh "https://raw.githubusercontent.com/mzet-/linux-exploit-suggester/master/linux-exploit-suggester.sh"
+  wget -q -O ./linux/linux-exploit-suggester.sh "https://raw.githubusercontent.com/mzet-/linux-exploit-suggester/master/linux-exploit-suggester.sh"
 }
 
 function add_mimikatz() {
   info "Downloading mimikatz"
   chkfs "./windows/"
-  wget -nv -O ./windows/mimikatz_trunk.zip "$(curl -s https://github.com/gentilkiwi/mimikatz/releases/latest | grep -o '"[^"]*"' | tr -d '"' | sed 's/tag/download/')/mimikatz_trunk.zip"
+  wget -q -O ./windows/mimikatz_trunk.zip "https://github.com/gentilkiwi/mimikatz/releases/latest/download/mimikatz_trunk.zip"
   unzip -q -o -d ./windows/mimikatz/ ./windows/mimikatz_trunk.zip
   rm ./windows/mimikatz_trunk.zip
 }
 
 function add_sharphound() {
   info "Downloading SharpHound"
   chkfs "./windows/"
-  wget -nv -O ./windows/SharpHound.exe "https://raw.githubusercontent.com/BloodHoundAD/BloodHound/master/Collectors/SharpHound.exe"
+  wget -q -O ./windows/SharpHound.exe "https://raw.githubusercontent.com/BloodHoundAD/BloodHound/master/Collectors/SharpHound.exe"
 }
 
 function add_juicypotato() {
   info "Downloading JuicyPotato"
   chkfs "./windows/"
-  wget -nv -O ./windows/JuicyPotato.exe "$(curl -s https://github.com/ohpe/juicy-potato/releases/latest | grep -o '"[^"]*"' | tr -d '"' | sed 's/tag/download/')/JuicyPotato.exe"
+  wget -q -O ./windows/JuicyPotato.exe "https://github.com/ohpe/juicy-potato/releases/latest/download/JuicyPotato.exe"
 }
 
 function add_nc() {
   info "Downloading nc for Windows"
   chkfs "./windows/"
-  wget -nv -O ./windows/nc.exe "https://gitlab.com/onemask/pentest-tools/-/raw/master/windows/nc.exe"
-  # chkfs "./linux/"
-  # todo add nc.traditional for linux
+  wget -q -O ./windows/nc.exe "https://gitlab.com/onemask/pentest-tools/-/raw/master/windows/nc.exe"
+  info "Downloading nc for Linux"
+  chkfs "./linux/"
+  wget -q -O ./linux/nc "http://www.stearns.org/nc/nc-static"
```

Comment:
This version of nc doesn't include the `-e` argument to specify the file to run after connection. In my opinion, we should feature the traditional netcat that has that feature ([nc.traditional](https://www.commandlinux.com/man-page/man1/nc.traditional.1.html)). Could probably be downloaded from [there](https://pkgs.org/download/netcat-traditional).

Reply:
It is not a problem if it is a deb package instead of a binary ?

Reply:
One of the use case of this tool is to upload it on a linux server to create a reverse shell, a deb package can be installed in exegol-images but it is not really useful..
Maybe this one instead ? https://github.com/andrew-d/static-binaries/blob/master/binaries/linux/x86_64/ncat

Reply:
Agreed, the .deb will be useless in this scenario. But on the other hand, a netcat binary that doesn't feature the `-e` argument is not that useful neither. @Dramelac did you make sure the ncat you linked ha that option? (can't test it right now)

Reply:
Yes it has the -e option